syntax = "proto3";

package user;

service UserService {
  rpc Login(LoginRequest) returns (LoginResponse);
}

message LoginRequest {
  string email = 1;
  string hashed_password = 2;
}

message LoginResponse {
  string identity_token = 1;
  string access_token = 2;
  string refresh_token = 3;
}
